metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -245,6 +245,56 @@ class ViewField:
     upstream_fields: List[str] = dataclasses_field(default_factory=list)
 
 
+@dataclass
+class ViewFieldNameInExplore:
+    explore: LookmlModelExplore
+    field: LookmlModelExploreField
+
+    def _form_field_name(self):
+        assert self.field.name is not None
+
+        if len(self.field.name.split(".")) != 2:
+            return self.field.name  # Inconsistent info received
+
+        view_name: Optional[str] = self.explore.name
+
+        if (
+            self.field.original_view is not None
+        ):  # if `from` is used in explore then original_view is pointing to
+            # lookml view
+            view_name = self.field.original_view
+
+        field_name = self.field.name.split(".")[1]
+
+        return f"{view_name}.{field_name}"
+
+    def name(self) -> str:
```

Comment:
is this the field's name or the upstream field's name?

Reply:
Let me rename it, it is upstream field name for explore field